examples/step-71/doc/results.dox

```diff
@@ -201,3 +290,220 @@ make sense:
+    </tr>
+  </table>
+  We can see in this case, convergence rates satisfy theoretical expectations.
+  Comparing taking $\gamma$ and taking $2\gamma$ in the `boundary_worker`,
+  $2\gamma$ converges better.
+
+
+  <h3>Test results on <i>Q<sub>3</sub></i> with <i>&gamma; = p(p+1)</i> </h3>
+
+  <table align="center" class="doxtable">
+    <tr>
+     <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H^1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H^2}$ </th><th> Conv. rates </th>
+    </tr>
+    <tr>
+     <td>   2                  </td><td>    2.045e-04 </td><td>       </td><td>   4.402e-03   </td><td>           </td><td> 1.641e-01 </td><td>   </td>
+    </tr>
+    <tr>
+     <td>   3                  </td><td>   1.312e-05   </td><td> 3.96  </td><td>  5.537e-04  </td><td>   2.99     </td><td> 4.096e-02 </td><td>  2.00  </td>
+    </tr>
+    <tr>
+     <td>   4                  </td><td>   8.239e-07 </td><td>  3.99  </td><td> 6.904e-05   </td><td> 3.00     </td><td> 1.023e-02 </td><td> 2.00 </td>
+    </tr>
+    <tr>
+     <td>   5                  </td><td>   5.158e-08  </td><td>  3.99 </td><td> 8.621e-06 </td><td>  3.00      </td><td> 2.558e-03  </td><td>  2.00  </td>
+    </tr>
+  </table>
+  We can see that the $L_2$ norm convergence rates are around 4,
+  $H^1$-seminorm convergence rates are around 3,
+  and $H^2$-seminorm convergence rates are around 2.
+  Results of this case are pretty close to the case with $2\gamma$ in `boundary_worker`.
+
+
+  <h3>Test results on <i>Q<sub>4</sub></i> with <i>&gamma; = p(p+1)</i> </h3>
+
+  <table align="center" class="doxtable">
+    <tr>
+     <th>Number of refinements </th><th>  $\|u-u_h^\circ\|_{L_2}$ </th><th>  Conv. rates  </th><th>  $|u-u_h|_{H^1}$ </th><th> Conv. rates </th><th> $|u-u_h|_{H^2}$ </th><th> Conv. rates </th>
+    </tr>
+    <tr>
+     <td>   2                  </td><td>    6.510e-06 </td><td>       </td><td> 2.215e-04   </td><td>           </td><td>  1.275e-02 </td><td>   </td>
+    </tr>
+    <tr>
+     <td>   3                  </td><td>   2.679e-07  </td><td>  4.60  </td><td> 1.569e-05  </td><td>   3.81    </td><td> 1.496e-03 </td><td>  3.09  </td>
+    </tr>
+    <tr>
+     <td>   4                  </td><td>   9.404e-09  </td><td> 4.83   </td><td> 1.040e-06    </td><td> 3.91       </td><td> 1.774e-04 </td><td> 3.07 </td>
+    </tr>
+    <tr>
+     <td>   5                  </td><td>   7.943e-10 </td><td>  3.56  </td><td>   6.693e-08 </td><td> 3.95     </td><td> 2.150e-05  </td><td> 3.04    </td>
+    </tr>
+  </table>
+  We can see that the $L_2$ norm convergence rates are around 5,
+  $H^1$-seminorm convergence rates are around 4,
+  and $H^2$-seminorm convergence rates are around 3.
+  The $L_2$ norm convergence rate in this case is the same as taking $2\gamma$ previously.
```

Comment:
Yes, so again the results are very much comparable. It is not necessary to spend a page of tables on *data* that has little actual *information*.